Fix CSS overrides in applications (#12646)
decidim/decidim: docs/modules/customize/pages/styles.adoc

Reviewer: We're already mentioning `decidim_application.scss` in line 57. Can you add a mention in that line to 

> 1. if you want to redefine a specific snippet of CSS (say a few classes but not a whole file) you can use decidim_application.scss. You can read more about this in the *Overriding CSS and styles* section

Reviewer: This is still missing @alecslupu

You can override the default configuration by creating a file  `config/tailwind.config.js.erb` starting from the following https://github.com/decidim/decidim/blob/develop/decidim-core/lib/decidim/assets/tailwind/tailwind.config.js.erb[content]

== Overriding CSS and styles

If you want to override a specific CSS class, you can do so by adding a new CSS class in your `app/assets/stylesheets/decidim_application.scss` file. This will allow you to override the default styles provided by Decidim.

*Please note:* the above file is created by default when you install Decidim. Also the file is empty. This file allows you to edit *only* the end user styles.

> If you want to edit the admin styles, you will need to create a new file called `app/packs/stylesheets/decidim/admin/decidim_application.scss`.
> 
> Additionally, you can also override the styles in the system panel by creating a new file called `app/packs/stylesheets/decidim/system/decidim_application.scss`.

Reviewer: ```suggestion
If you want to override other layouts, you can do so by creating new files: 

. /admin: `app/packs/stylesheets/decidim/admin/decidim_application.scss`
. /system: `app/packs/stylesheets/decidim/system/decidim_application.scss`
. /design: `app/packs/stylesheets/decidim/design/decidim_application.scss`
```

Author: Note: That we do not add Override for design application, but we just import the core override into design ( so that the developer to see in the design app how things are fall in based one their changes )

You can override the default configuration by creating a file  `config/tailwind.config.js.erb` starting from the following https://github.com/decidim/decidim/blob/develop/decidim-core/lib/decidim/assets/tailwind/tailwind.config.js.erb[content]

== Overriding CSS and styles

If you want to override a specific CSS class, you can do so by adding a new CSS class in your `app/assets/stylesheets/decidim_application.scss` file. This will allow you to override the default styles provided by Decidim.

*Please note:* the above file is created by default when you install Decidim. Also the file is empty. This file allows you to edit *only* the end user styles.

> If you want to override other layouts, you can do so by creating new files: 

Reviewer: There's a trailing whitespace here

```suggestion
If you want to override other layouts, you can do so by creating new files:
```
